share/functions/export.fish

```diff
@@ -9,7 +9,11 @@ function export --description 'Set global variable. Alias for set -gx, made for
             case 1
                 set -gx $v $$v
             case 2
-                set -gx $v[1] $v[2]
+                if test $v[1] = PATH
```

Comment:
This should be used for $PATH, $CDPATH and $MANPATH - the hardcoded list of special variables.